### DISCUSSION TITLE
Welcome to gpu-io Discussions!

### DISCUSSION THREAD

**Discussion by amandaghassaei** [amandaghassaei/glcompute#4] [Announcements]
## 👋 Welcome!

I'm opening up discussions to let people ask questions about how to use gpu-io without formally submitting an issue.  Feel free to create new threads to ask questions and share ideas/things you've made.

Have you made something with gpu-io?  Add it to the [Show and Tell](https://github.com/amandaghassaei/gpu-io/discussions/5) thread.

Thanks!
Amanda

**Comment by gormonn** [comment]
@amandaghassaei Hello!
I just want to say it looks great.
